src/Microsoft.DotNet.Cli.Sln.Internal/SlnFile.cs

```diff
@@ -698,17 +698,34 @@ internal void ReadLine(string line, int currentLine)
 
         internal void Write(TextWriter writer)
         {
-            foreach (DictionaryEntry e in _values)
+            if (_isMetadata)
             {
-                if (!_isMetadata)
+                foreach (DictionaryEntry e in _values)
                 {
-                    writer.Write("\t\t");
+                    if (Id != null)
+                    {
+                        writer.Write(Id + ".");
+                    }
+                    writer.WriteLine(e.Key + " = " + e.Value);
```

Comment:
I'm going to revert this file since we are keeping it low level. The code I added was to ensure build configurations appear in alphabetical order but I won't worry about that and will let VS fix that up.